### PR TITLE
Add missing migrations on Django 1.10

### PR DESCRIPTION
**Pull request by giginet**
When I executed `makemigrations` in Django 1.10 environment, some migrations have been missing.

### CODE REVIEW

File: tests/settings.py [hunk 1]
```diff
@@ -111,6 +111,9 @@
                 os.path.join(os.path.dirname(__file__), 'templates'),
             ],
             'OPTIONS': {
+                'context_processors': [
+                    'django.contrib.auth.context_processors.auth',
```
Comment (giginet) [inline tests/settings.py]: It is needed or else it is raised the following error.

```
?: (admin.E402) 'django.contrib.auth.context_processors.auth' must be in TEMPLATES in order to use the admin application.
```

**Comment by lambdalisue** [comment]
It added default supplement which is not required for general user so I won't merge this.

**Comment by giginet** [comment]
OK, this migration is not required, However, `context_processor` is needed. 
So I should to fix only a `settings.py`

**Comment by giginet** [comment]
I thought again. your comment is not correct.

> It added default supplement which is not required for general user so I won't merge this.

This migration is for `RegistrationProfile` not `RegistrationSupplement`.
So it is needed.

However, `dependencies` is wrong.
`0002_supplement` is not related with this migration, so I changed.

File: src/registration/migrations/0003_registration_profile_status.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('registration', '0002_default_supplement'),
```
Comment (lambdalisue) [inline src/registration/migrations/0003_registration_profile_status.py]: **NOTE**
It depends on `0002_default_supplement` but actual migration won't performed so it's ok. https://github.com/lambdalisue/django-inspectional-registration/blob/master/src/registration/migrations/0002_default_supplement.py#L14

**Comment by lambdalisue** [comment]
:tada:

**Comment by giginet** [comment]
Thank you 👯

**Pull request merged**